publiq-ui/pub-checkbox.vue

```diff
@@ -1,12 +1,33 @@
 <template>
-  <b-form-checkbox :checked="checked" :disabled="disabled" @input="handleInput">
-    <slot />
-  </b-form-checkbox>
+  <div class="container-checkbox">
+    <input
+      :id="id"
+      type="checkbox"
+      :name="name"
```

Comment:
Do we need both the `id` and `name`?

Reply:
They serve different purposes. Traditionally name is something you have to pass to a checkbox input to then later get the selected values by name. But in vue we could leave it, I think

Reply:
I'll leave the name prop, which is optional. If we need it it's there